Bugfixing for January 2026 Patch

ilkober opened the pull request:
All the necessary (and not) bugfixes and tweaks

Thinking-waffle committed:
Update tibet.txt

Thinking-waffle committed:
Update 323-Nepal.txt

Thinking-waffle committed:
Update r56_lithuania.txt

Thinking-waffle committed:
Update GER_l_english.yml

katsuragiyy committed:
VEM Portraits by Harry

Commit:
Adding a Dam

Commit:
smol buff

Commit:
a few more eng event fixes

Commit:
India krap

Commit:
Removing Obsolete content

Since NSB 4 years ago

Commit:
Localisation fix

Commit:
portrait fix

Commit:
random interface typos fixed

Commit:
Fixing bug from the GoE integration

Commit:
Vichy France Cosmetic Tag Fix

More than cosmetic, this fixed a few triggers here and there

Commit:
Fixing up Mongolia

Commit:
Palestine Fixed Up

Commit:
Congo fixes

Commit:
Removing bad RT56 pre-MTG portraits for HOL

Commit:
Advisors for HOL I forgor

Commit:
Removing obsolete GER focus tree mentions

Commit:
Loc Fix for PAL

Commit:
A few USSR event fixes

Commit:
Removing a province from a states list

Commit:
Removed a useless hidden SUD focus

ilkober committed:
Leader creation tweaks 1

Removed Brazilian leader election featuring characters from 50s, 60s and 70s - info is still there for those interested
Removed some old instances of leader creation across several history files
Converted 3-4 instances of leader creation to characters

marijn211 committed:
6th research slot focus for anti-USA Philippines

marijn211 committed:
Update countries for which Diversify Special Forces is blocked

marijn211 committed:
Fix some missing ranger localization

marijn211 committed:
Edge case bypass for Philippines

katsuragiyy committed:
changed the name of north borneo state (916) to something more appropriate

e-ee-ee committed:
porting vanilla AI changes to japan in response to shorter focus times

e-ee-ee committed:
made it somewhat easier to integrate puppets

e-ee-ee committed:
removed/reduced the stab malueses on confucian fascism and the power of confucious

e-ee-ee committed:
Buffed the Chinese United Front

e-ee-ee committed:
fixed CUF effects

marijn211 committed:
Fix missing brackets in China edits

e-ee-ee committed:
Various Tweaks and bugfixes

marijn211 committed:
Slightly polish Molotov Ribbentrop focus issues

marijn211 committed:
Fix loc string that vanilla replaced

marijn211 committed:
Fix two bugs

e-ee-ee committed:
updated Japan's tree + manchu bugfix

e-ee-ee committed:
fixed japan civil war and dailian

e-ee-ee committed:
missed south sakalin

katsuragiyy committed:
ported PHI checks to economy and manpower laws

katsuragiyy committed:
fix desert tiles in maphilindo

marijn211 committed:
Fix duplicate admiral for PHI

marijn211 committed:
Fix leader for PHI monarchist path

marijn211 committed:
Adjust PHI path for early independence

e-ee-ee committed:
Fixed a bug where fascist warlords would be puppeted by the RoC

e-ee-ee committed:
updated RoC border wars

Commit:
Fixing a few Machu related bugs

Commit:
Create manchukuo_TSR.txt

Commit:
Fixing AI for Machu + Few Yoinked Warlords

Commit:
Japanese AI fixes

Commit:
Fixed dozens of decisions for China

e-ee-ee committed:
fixed a loc issue

marijn211 committed:
Fix some unported lines in SOV events

marijn211 committed:
Tiny focus filter fix

marijn211 committed:
Fix lacking army chief choice for Japan

marijn211 committed:
Fix missing focus check for JAP army chief

marijn211 committed:
Allow Japan to edit its bicycle template

marijn211 commented:
@ilkober Could you greatly limit border wars for Mongolia? Especially for AI Mongolia. Its really annoying when you're playing Japan

marijn211 committed:
Fix fall of Nanjing on capturing Shanghai

marijn211 committed:
Slightly tweak extremely low tension gain from peacedeals

marijn211 committed:
Fix unreliable AI for non-NSB alt-hist SOV

marijn211 committed:
Old tree Trotsky gets ideology spread faction manifest

marijn211 committed:
Polish industry companies for old Soviet tree

marijn211 committed:
Restore old world tension from war declaring

marijn211 committed:
Disable Mongolian border wars as dominion while that steals Soviet troops

marijn211 committed:
Update PHI with missing tag targets

marijn211 committed:
Fix localization issue

BiccyThiccy committed:
Adding 2/3 the global continental arrays from NCNS

The middle east one is throwing errors for whatever reason so that one can be looked into another time

BiccyThiccy committed:
NCNS - Incorporating new traits from 00_traits.txt

BiccyThiccy commented:
@e-ee-ee I think one of your commits has added an error regarding the new shared branch

katsuragiyy committed:
PRC tooltip fix

okay guys its time to create my own proper branch for the first time in like 3 months lmao

Commit:
some ITA fixes

Commit:
ALG fixes

Commit:
Update AAT_Iceland.txt

Commit:
CHL + BUL fixing

Commit:
Update ARM_events.txt

Commit:
cleaned up on_actions

Commit:
fixing chinese on actions

Commit:
Fixing some AI behavior stuff for CHI

Commit:
reducing log

Commit:
improving icons

Commit:
Update japan.txt

Commit:
Update TSR_lingguang_incident_joint_branch.txt

marijn211 committed:
Clean up code of another commit on this MR

marijn211 commented on localisation/english/TIB_l_english.yml:
```diff
@@ -439,8 +439,8 @@
 
  himalaya_tibet.44.t:0 "Stalemate at [434.GetName]"
- himalaya_tibet.46.desc:0 "Our military forces have not been able to advance in the battle of [434.GetName]. We have fended off many enemy attacks but our offensive charges and morale were not good enough to push through the enemy lines. However, there is still much that we can learn and information we can gather from these skirmishes..."
- himalaya_tibet.46.a:0 "A waste of blood and tears..."
```
This still seems broken, this probably doesn't have to be undone, but in the events files, there seem to be several references to non-existent loc that need to be worked away

marijn211 commented on common/national_focus/sudan.txt:
Can you elaborate @Xaelorichero-02 ?

marijn211 replied:
Ah hidden in that sense, good to know

marijn211 commented:
Other than those 2 comments the diff seems fine

Commit:
tib quick fix

Commit:
Merge branch 'Beta-Edits' into bugfixing-for-jan26

Commit:
Update r56_traits.txt

TheBrit7 committed:
Merge branch 'Beta-Edits' into bugfixing-for-jan26

TheBrit7 committed:
Fixes

marijn211 reviewed (approved):
lll try include a few lastminute fixes and then merge to make place for a new bugfix PR

marijn211 committed:
Fix broken loc after MIO trait typo fix

marijn211 committed:
Text overlap less often

marijn211 committed:
Fix error I caused

marijn211 committed:
Fix Tender Loving Empire becoming nameless

marijn211 committed:
Fix 2 errors about duplicate MIO

marijn211 committed:
Add plane design to USA that China was referencing

marijn211 committed:
Port CHI DLC variant of MIO and link to tree choice global flag

marijn211 committed:
Fix error-causing typo

marijn211 committed:
Resolve more issues with MIO
